wandb/sdk/launch/deploys/aws-eks/launch-config.yaml

```diff
@@ -73,12 +73,21 @@ roleRef:
 ---
 apiVersion: v1
 data:
-  wandb-base-url: "https://api.wandb.ai" ### TODO: SET YOUR WANDB BASE URL
-  wandb-project: "launch_test" # TODO: SET YOUR ML PROJECT
-  max-jobs: -1 # TODO: SET YOUR MAX CONCURRENT JOBS HERE
+  launch-config.yaml: |
+    base_url: https://api.wandb.ai # TODO: SET YOUR WANDB BASE URL
+    project: launch-welcome # TODO: SET YOUR ML PROJECT
+    max_jobs: -1 # TODO: SET YOUR MAX CONCURRENT JOBS HERE
+    registry:
+      url:  # TODO: SET ECR REPOSITORY URL HERE
+      ecr-provider: AWS # TODO: SET ECR PROVIDER HERE
+    build:
+      type: kaniko
+      cloud-provider: AWS # TODO: SET CLOUD PROVIDER HERE
```

Comment:
We should think about the on premise case which will likely use minio.  I suppose it might work as AWS, not something for MVP but customers will want it relatively soon.

Reply:
I'll file a ticket for that, it will take some research on me end, I have limited experience with minio

Reply:
Minio literally uses the same API as AWS S3, we just need to pipe through a custom domain.  I would imagine there's docs for how to get kaniko to do this somewhere.  Happy to work with you when we go down that path.

Reply:
Sounds great! That will be a nice alternative.